src/whylogs/core/columnprofile.py

```diff
@@ -4,10 +4,12 @@
 from whylogs.core.statistics import CountersTracker, NumberTracker, SchemaTracker
 from whylogs.core.statistics.datatypes import StringTracker
 from whylogs.core.statistics.hllsketch import HllSketch
+from whylogs.core.statistics.constraints import ValueConstraints, SummaryConstraints, SummaryConstraint
 from whylogs.core.types import TypedDataConverter
-from whylogs.proto import ColumnMessage, ColumnSummary, InferredType
+from whylogs.proto import ColumnMessage, ColumnSummary, InferredType, Op
```

Comment:
are these already merged in proto ?

Reply:
Not merged, but in a PR
https://github.com/whylabs/whylogs-proto/pull/4

Reply:
FYI I merged the protobuf change. Submodule is annoying in many ways...